### PR TITLE
Add/support ios/Net/Android log

### PR DESCRIPTION
**Pull request by mgcouchbase**
#### Fixes #.

- [x] Ran `flake8`
- [x] Ran `run_repo_tests.sh`

#### Changes proposed in this pull request:
net log
- http://uberjenkins.sc.couchbase.com:8080/job/CBLITE_DOTNET-Net-windows-TestSever-Functional-DELTA-SYNC-tests/95/artifact/results/logs/test_delta_sync_nested_doc%5B1-pull%5D.zip/*view*/
- 
-http://uberjenkins.sc.couchbase.com:8080/job/CBLITE_iOS-Listener-TestServer-Functional-SSL-DELTA-SYNC-tests/96/

retested after the change and it looks good: http://uberjenkins.sc.couchbase.com:8080/job/CBLITE_iOS-Listener-TestServer-Functional-SSL-DELTA-SYNC-tests/97/artifact/results/logs/

### CODE REVIEW

File: testsuites/CBLTester/CBL_Functional_tests/conftest.py [hunk 1]
```diff
@@ -539,13 +537,15 @@ def params_from_base_suite_setup(request):
             if test.rep_call.failed:
                 failed_test_list.append(test.rep_call.nodeid)
         zip_data = suite_cbllog.get_logs_in_zip()
-        suite_log_zip_file = "Suite_test_log_{}.zip".format(str(time.time()))
+        suite_log_zip_file = "Suite_test_log_{}.zip"
```
Comment (euniceatcouchbase) [inline testsuites/CBLTester/CBL_Functional_tests/conftest.py:540]: do you intent to replace {} somewhere? didn't find it. if not, the file name doesn't have to contain {}, which causes confusion.

**Review by euniceatcouchbase — CHANGES_REQUESTED** [review]

**Review by euniceatcouchbase — APPROVED** [review]

File: testsuites/CBLTester/CBL_Functional_tests/conftest.py [hunk 2]
```diff
@@ -539,13 +537,15 @@ def params_from_base_suite_setup(request):
             if test.rep_call.failed:
                 failed_test_list.append(test.rep_call.nodeid)
         zip_data = suite_cbllog.get_logs_in_zip()
-        suite_log_zip_file = "Suite_test_log_{}.zip".format(str(time.time()))
+        suite_log_zip_file = "Suite_test_log.zip"
```
Comment (sridevi-15) [inline testsuites/CBLTester/CBL_Functional_tests/conftest.py:540]: having time stamp will be helpful, can you add the time stamp back?

**Review by sridevi-15 — CHANGES_REQUESTED** [review]
looks good to me, but a minor one to add the timestamp back

File: testsuites/CBLTester/CBL_Functional_tests/conftest.py (hunk 2 above)
Comment (mgcouchbase) [reply testsuites/CBLTester/CBL_Functional_tests/conftest.py:540]: We have the logic for archiving logs for test level as well as test suite level, I added the logic that if logs already archived in the test level I do not archive at the suite level. If we add the time stamp we end up archiving twice .

We do not need it, we have a unique test name to differentiate the test logs.

**Review by sridevi-15 — APPROVED** [review]

**Pull request merged**